tdrs-backend/tdpservice/parsers/schema_defs/header.py

```diff
@@ -68,11 +68,15 @@
             endIndex=14,
             required=True,
             validators=[
-                validators.oneOf(("01", "02", "04", "05", "06", "08", "09", "10", "11", "12", "13", "15",
-                                  "16", "17", "18", "19", "20", "21", "22", "23", "24", "25", "26", "27",
-                                  "28", "29", "30", "31", "32", "33", "34", "35", "36", "37", "38", "39",
-                                  "40", "41", "42", "44", "45", "46", "47", "48", "49", "50", "51", "53",
-                                  "54", "55", "56", "66", "72", "78"))
+                validators.or_validators(
+                    validators.isInStringRange(0, 2),
```

Comment:
we also need to allow spaces for `state_fips` (for tribes) as well as spaces for `tribe_code` (for states and territories).